### PR TITLE
Complete test cases

### PR DESCRIPTION
**Pull request by erandiganepola**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: tests/test.bal [hunk 1]
```diff
@@ -561,8 +851,8 @@ function setConfParams (string|null confParam) returns string {
             return param;
         }
     null => {
-            io:println("Empty value!");
-            return "";
+    io:println("Empty value!");
```
Comment (keerthu) [inline tests/test.bal]: Format these lines.

**Comment by keerthu** [comment]
Add .ballerina folder to package-salesforce. Otherwise user will get "cannot find package 'salesforce'" exception.

File: salesforce/tests/test.bal [hunk 2]
```diff
@@ -72,25 +87,152 @@ function testGetOrganizationLimits () {
...
             test:assertFail(msg = err.messages[0]);
         }
     }
 }
 
+//=============================== Query ==================================//
+
 @test:Config
 function testGetQueryResult () {
     io:println("\n-------------------------- getQueryResult () -------------------------");
```
Comment (muthulee) [inline salesforce/tests/test.bal]: Please remve io:println. Move it to a log.

File: salesforce/tests/test.bal [hunk 3]
```diff
@@ -152,19 +298,52 @@ function testSObjectPlatformAction () {
...
-        sf:SalesforceConnectorError err => {
+        SalesforceConnectorError err => {
+            test:assertFail(msg = err.messages[0]);
+        }
+    }
+}
+
+@test:Config
+function testDescribeAvailableObjects () {
+    io:println("\n----------------------- describeAvailableObjects() ---------------------------");
```
Comment (muthulee) [inline salesforce/tests/test.bal]: Remove io:println from all Test cases.

**Pull request closed without merging**